### PR TITLE
Geolocate endpoint and tests

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/intel/ITIPIntelTest.java [hunk 1]
```diff
@@ -121,6 +122,102 @@ public void testIpLookupMalicious_8() throws PangeaException, PangeaAPIException
         assertNotNull(response.getResult().getRawData());
     }
 
+    @Test
+    public void testIpGeolocate_1() throws PangeaException, PangeaAPIException {
+        // Default provider, not verbose by default, not raw by default;
+        IpGeolocateResponse response = client.geolocate("93.231.182.110");
```
Comment (doubletooth) [inline packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/intel/ITIPIntelTest.java:128]: do we want to make `93.231.182.110` a class level variable so you don't have to change 12 locations if we ever have to change the IP?

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/intel/IpGeolocateData.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package cloud.pangeacyber.pangea.intel;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class IpGeolocateData {
```
Comment (doubletooth) [inline packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/intel/IpGeolocateData.java:7]: Would this be analogous to a dataclass in python? 

also completely other random question. do we want to have helpers that pretty print this data or something like that?

**Review by doubletooth — APPROVED** [review]

File: packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/intel/ITIPIntelTest.java (hunk 1 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/intel/ITIPIntelTest.java:128]: No sure about your question. In which class would so save the IP? in IpGeolocateResponse? I don't get the purpose of that

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/intel/IpGeolocateData.java (hunk 2 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/intel/IpGeolocateData.java:7]: Yes, sort of I think. How would you do it?
Totally agree about the pretty print, I've thought about it to implement in API errors, but I didn't do it yet

Comment (doubletooth) [reply packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/intel/IpGeolocateData.java:7]: we can file a ticket for this down the line or if a customer asks for it

**Pull request merged**